### PR TITLE
Windows MSVC

### PR DESCRIPTION
**Pull request by diorcety**
Patche set for compiling the plugin on Windows

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -247,17 +263,20 @@ def finalize_options(self):
...
+                openmpflag = '-fopenmp'
+                compileflags = COMPILE_FLAGS
             for e in self.extensions:
-                if '-fopenmp' not in e.extra_compile_args:
-                    e.extra_compile_args += ['-fopenmp']
-                if '-fopenmp' not in e.extra_link_args:
-                    e.extra_link_args += ['-fopenmp']
+                if openmpflag not in e.extra_compile_args:
+                    e.extra_compile_args += [openmpflag]
+                    e.extra_compile_args += compileflags
```
Comment (kiyo-masui) [inline setup.py]: `compileflags` only gets added to `e.extra_compile_args` if compiling with openmp. Maybe move to the `_needs_stub` loop, but be careful that they only get added once (I seem to remember `build_extensions` being called multiple times being an issue.

File: src/bitshuffle_core.c [hunk 2]
```diff
@@ -1138,7 +1138,7 @@ int64_t bshuf_untrans_bit_elem(const void* in, void* out, const size_t size,
 int64_t bshuf_blocked_wrap_fun(bshufBlockFunDef fun, const void* in, void* out, \
         const size_t size, const size_t elem_size, size_t block_size) {
 
-    size_t ii;
+    long long ii;
```
Comment (kiyo-masui) [inline src/bitshuffle_core.c]: Why?

File: setup.py [hunk 3]
```diff
@@ -19,7 +19,7 @@
 VERSION_POINT = 4
 
 # Only unset in the 'release' branch and in tags.
-VERSION_DEV = 1
```
Comment (kiyo-masui) [inline setup.py]: Only allowed in the release branch. Please change back.

File: src/bitshuffle_core.c (hunk 2 above)
Comment (diorcety) [reply src/bitshuffle_core.c]: MSVC openmp doesn't like the size_t type

Comment (kiyo-masui) [reply src/bitshuffle_core.c]: But it is used throughout the code and is a very standard type. Why only this one place?

Comment (diorcety) [reply src/bitshuffle_core.c]: The openmp pragma (as iterator)

Comment (kiyo-masui) [reply src/bitshuffle_core.c]: Okay fair enough. Maybe state this in a comment above this line so it doesn't bewilder future developers.

**Pull request merged**